extensions/amp-date-display/0.1/amp-date-display.js

```diff
@@ -185,13 +224,44 @@ export class AmpDateDisplay extends AMP.BaseElement {
     return epoch;
   }
 
+  /**
+   * @param {!Element} element
+   * @return {Object<string, *>|undefined}
+   * @private
+   */
+  parseLocalOptionsAttrs_(element) {
+    const getCamelCase = (kebabCase) =>
+      kebabCase
+        .split('-')
+        .map((str, index) =>
+          index === 0 ? str : str[0].toUpperCase() + str.substring(1)
+        )
+        .join('');
+
+    const localeOptions = SUPPORTED_LOCALE_OPTIONS_ATTRS.filter((attr) =>
+      element.hasAttribute(`data-options-${attr}`)
+    );
+    if (localeOptions.length === 0) {
+      return undefined;
+    }
+
+    return localeOptions.reduce(
+      (options, attr) => ({
+        ...options,
+        [getCamelCase(attr)]: element.getAttribute(`data-options-${attr}`),
+      }),
+      {}
+    );
```

Comment:
WDYT about following the logic that exists for `attrPrefix`?https://github.com/ampproject/amphtml/blob/ebff027e9f4ef31e667cb8942ffa387171f33c8f/src/preact/base-element.js#L1174-L1184

Reply:
Sure, it's good to make the logic of the 2 versions similar. Updated.